src/Chapter1.hs

```diff
@@ -631,8 +647,16 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int
+sumLast2 n = modDigits n
+  where
+    modDigits :: Int -> Int
+    modDigits x
+      | x < 10 = x
+      | x < 100 = mod x 10
+      | x < 1000 = mod x 100
+      | otherwise = mod x 1000
```

Comment:
You are on the right way here! You see that there is the pattern in all of this steps.
Instead of manual few steps here, you can continue doing `mod x 10` here until x would be smaller than 10 :)
For that you can use recursiuon (call the same function over and over again from the body of the function itself 👌🏼